src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -32,6 +31,119 @@ internal void VerifyDiagnostics(string source, CSharpCompilationOptions options,
     public class LocalFunctionTests : LocalFunctionsTestBase
     {
         [Fact]
+        public void LocalFunctionAttributeOnTypeParameter()
+        {
+            var tree = SyntaxFactory.ParseSyntaxTree(@"
+using System;
+class C
+{
+    public void M()
+    {
+        void Local<[A, B, CLSCompliant, D]T>() { }
+        Local<int>();
+    }
+}");
+            var comp = CreateCompilationWithMscorlib(tree);
+            comp.DeclarationDiagnostics.Verify();
```

Comment:
>comp.DeclarationDiagnostics.Verify(); [](start = 12, length = 37)

Could you also assert this at the end of the test to make sure examining symbols doesn't add anything? Same for the other test.